src/Aquifer.Data/Entities/FeedbackEntity.cs

```diff
@@ -8,7 +8,15 @@ public class FeedbackEntity
     public string? UserId { get; set; }
     public string? Phone { get; set; }
     public string Feedback { get; set; } = null!;
+    public FeedbackType FeedbackType { get; set; }
 
     [SqlDefaultValue("getutcdate()")]
     public DateTime Created { get; set; } = DateTime.UtcNow;
+}
+
+public enum FeedbackType
+{
+    None = 0,
+    WellContactForm = 1,
+    WellContactPrompt = 2
```

Comment:
I'm not sure I understand the different between these. Should it be something like `WellContactOnly` and `WellFullFeedback` or something to distinguish really clearly?

Reply:
My thinking was that one is the form they have to manually go to, while the other one is the prompt that comes up. ContactOnly and FullFeedback seems vague to me also, since we could decide to add more to the prompt if we wanted. I'm not really sure what good names for these are, they're so similar, and could potentially get more overlap as time goes by.

Reply:
If we're trying to differentiate auto vs user-triggered rather than what the feedback is inside them, it almost seems like we should call it `FeedbackTriggerType` and have `Auto` and `User`.

Reply:
I don't know about the word "Trigger", because a voluntarily filled form is not necessarily triggered. Also, a User fills out both something that's Auto and Volunteered.

Reply:
We're just trying to distinguish which feedback form it came from in the app. One the user has to manually navigate to, and now the new one that Elisabeth is working on is a popup. But aside from their location in the app, we shouldn't assume any differences between them, and we should assume we'll have other feedback forms in the future.

How do you both feel about `WellSettingsFeedbackForm` and `WellPopupFeedbackForm`? This shouldn't hold us up much more though, no one is seeing these enum names except us and there's nothing functional about the names, they can be changed later.

Reply:
The reqs also have contact type dropdown with a value. In the `ResourceContentVersionFeedback` table we have both of them as fields. 
- Are we okay with `ContactType` and `ContactValue` going in the JSON `Feedback` column? 
- Or would we want them to be new columns on the Feedback Table like the `ResourceContentVersionFeedback` table?
- Would it be better to have some sort of intermediary table handle that info?